utils/deletes.py

```diff
@@ -63,7 +66,7 @@ def examine(tweet):
 
         if tweet_status == TWEET_OK:
             return TWEET_OK
-        elif retweet is None:
+        elif retweet is None or tweet_status == TWEET_BLOCKED:
```

Comment:
I'm having trouble understanding the logic of what's going on here.

Reply:
Confusing to try to explain in terms of original, retweet, etc., so let's try this:
If X is a retweet of Y. If X is blocked, then report as blocked.
Otherwise, if X is not blocked then determine status based on Y.

Open to suggestions about how to make the code clearer.

Reply:
Yeah, I'm still having trouble understanding this. But since you are actively using this I'll merge it and we can improve it over time as needed.